qbft/generate.go

```diff
@@ -1,4 +1,4 @@
 package qbft
 
 //go:generate rm -f ./messages_encoding.go
-//go:generate go run github.com/ferranbt/fastssz/sszgen --path messages.go --include ./types.go,../types/signer.go,../types/operator.go --exclude-objs OperatorID
+//go:generate go run github.com/ferranbt/fastssz/sszgen --path messages.go --include ./types.go,../types/signer.go,../types/operator.go --exclude-objs OperatorID,ProcessingMessage
```

Comment:
we don't pass processing message on wire..
why do we need ssz encoding for it?

Reply:
I've added ProcessingMessage to the excluded objects